docs/guides/preinstall.md

```diff
@@ -1,15 +1,15 @@
 # Pre install steps
 
-These steps are common to most distros. Please refer to both this page and this wiki's install page for your distro.
+While the following steps are common to most distributions, it is recommended to check the [install page for your distro](https://wiki.t2linux.org/distributions/overview/) before following this guide.
```

Comment:
it is recommended to check the [install page for your distro](https://wiki.t2linux.org/distributions/overview/) ~before~ *as well* following this guide.

This seems to be a better option.